CoEpi/ui/RootWireframe.swift

```diff
@@ -22,11 +21,12 @@ class RootWireFrame {
         window.makeKeyAndVisible()
 
         self.homeViewController = homeViewController
-
-        if showOnboarding {
+        
+        if KeyValueStore().getBool(key: KVSBoolKey.seenOnboarding) != true {
```

Comment:
`KeyValueStore` should be an injectable component (this allows to change its implementation / mock it). So wire it in `Dependencies` and retrieve here with `container.resolve()` (or maybe pass it directly to the initializer, like the rest of components).